docs/products/kafka/karapace/concepts/acl-definition.rst

```diff
@@ -0,0 +1,79 @@
+
+ACLs definition
+================
+An ACL for Karapace schema registry authorization consists of zero or more entries that specify a **username**, an **operation**, and a **resource**. 
+
+* **Username** is the name of a service user in the Aiven for Apache Kafka® service.
+* **Operations** are:
+  
+  * ``schema_registry_read``
+  * ``schema_registry_write`` (always includes ``schema_registry_read``)
+
+*  A **resource** can be in the following formats:
+  
+   * ``Config:``: the entry controls access to global compatibility configurations. Karapace only allows a user to retrieve and change the default schema compatibility mode via the global ``Config:`` resource. For more information, see the project `README <https://github.com/aiven/karapace/blob/main/README.rst>`_.
+
+    .. Note::
```

Comment:
The `.. Note::` should be in line with the ``Config`` otherwise the following happens (see grey background of the node)

![image](https://user-images.githubusercontent.com/11272743/189310010-225360a9-08ec-4a09-8f00-d45e87b1a9e4.png)